app/assets/stylesheets/common/base/topic-post.scss

```diff
@@ -1,15 +1,54 @@
+// placeholder
+@keyframes placeHolderShimmer {
```

Comment:
Now that this is used in a couple of different places, can we move it somewhere more generic than `topic-post.scss`? We have a couple of other `@keyframes` in `common/base/discourse.scss`.